src/resources/views/inc/alerts.blade.php

```diff
@@ -13,7 +13,7 @@
             $(function(){
               new PNotify({
                 // title: 'Regular Notice',
-                text: "{{ $message }}",
+                text: "{!! str_replace('"', "'", $message) !!}",
```

Comment:
You sure you've checked this?

What if I have:

```
$message = '"';
```

Reply:
Yup. Just double-checked to be sure: https://cl.ly/3C2V1s0i1Y26